src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/toolwindow/TestGenieToolWindow.kt

```diff
@@ -1,49 +1,67 @@
 package nl.tudelft.ewi.se.ciselab.testgenie.toolwindow
 
-import com.intellij.openapi.ui.Messages
-
-import java.awt.Dimension
-import java.awt.GridBagLayout
-import java.awt.event.ActionEvent
-import javax.swing.JButton
 import com.intellij.openapi.ui.ComboBox
+import com.intellij.openapi.ui.Messages
 import com.intellij.ui.components.JBLabel
 import com.intellij.ui.components.JBScrollPane
 import com.intellij.util.ui.FormBuilder
+import com.intellij.util.ui.JBUI
+import com.intellij.util.ui.UIUtil
+import org.jdesktop.swingx.JXTitledSeparator
 import java.awt.*
+import java.awt.event.ActionEvent
 import javax.swing.*
 
 /**
  * This class stores the UI of the TestGenie tool window.
  */
 class TestGenieToolWindow {
 
-    private var searchBudget: JSpinner = JSpinner(SpinnerNumberModel(60, 0, 10000, 1))
-    private var localSearchBudgetType: ComboBox<LocalSearchBudgetType> = ComboBox(LocalSearchBudgetType.values())
-    private var localSearchBudgetValue: JSpinner = JSpinner(SpinnerNumberModel(5, 0, 10000, 1))
+    // UI elements for EvoSuite parameters
     private var stoppingCondition: ComboBox<StoppingCondition> = ComboBox<StoppingCondition>(StoppingCondition.values())
+    private var searchBudget: JSpinner = JSpinner(SpinnerNumberModel(60, 0, 10000, 1))
     private var initializationTimeout: JSpinner = JSpinner(SpinnerNumberModel(120, 0, 10000, 1))
     private var minimisationTimeout: JSpinner = JSpinner(SpinnerNumberModel(60, 0, 10000, 1))
     private var assertionTimeout: JSpinner = JSpinner(SpinnerNumberModel(60, 0, 10000, 1))
     private var junitCheckTimeout: JSpinner = JSpinner(SpinnerNumberModel(60, 0, 10000, 1))
-    private var population: JSpinner = JSpinner(SpinnerNumberModel(50, 0, 10000, 1))
     private var populationLimit: ComboBox<PopulationLimit> = ComboBox(PopulationLimit.values())
+    private var population: JSpinner = JSpinner(SpinnerNumberModel(50, 0, 10000, 1))
```

Comment:
Have we not agreed to move population to advanced settings (as per meeting mattermost post of client)?

Reply:
As we discussed in the morning. The clients mattermost is outdated and in your notes "population" is in the quick access settings. Although, this was a confusing point during the meeting, so I wouldn't be surprised if we need to change it later.